partiql-eval/src/lib.rs

```diff
@@ -95,6 +114,124 @@ mod tests {
         assert_eq!(result.len(), 3);
     }
 
+    #[test]
+    fn arithmetic() {
+        // Tests arithmetic ops using int, real, decimal, null, and missing with values defined from
+        // `data_arithmetic_tuple`
+        fn arithmetic_logical(binary_op: BinaryOp, lit: Value) -> BindingsExpr {
+            BindingsExpr::From(logical::From {
+                expr: ValueExpr::VarRef(BindingsName::CaseInsensitive("data".into())),
+                as_key: "data".to_string(),
+                at_key: None,
+                out: Box::new(BindingsExpr::Select(logical::Select {
+                    exprs: HashMap::from([(
+                        "b".to_string(),
+                        ValueExpr::BinaryExpr(
+                            binary_op,
+                            Box::new(ValueExpr::Path(
+                                Box::new(ValueExpr::VarRef(BindingsName::CaseInsensitive(
+                                    "data".into(),
+                                ))),
+                                vec![PathComponent::Key("a".to_string())],
+                            )),
+                            Box::new(ValueExpr::Lit(Box::new(lit))),
+                        ),
+                    )]),
+                    out: Box::new(BindingsExpr::Output),
+                })),
+            })
+        }
+        // Plan for `select a + <lit> as b from data`
```

Comment:
This test doesn't actually do any assertions. Currently just printing the evaluated output as a sanity check. I felt like the tests added to `partiql-value/src/lib.rs` were sufficient for this PR.

I can add assertions in an upcoming commit if that's desired.

Reply:
I think this is ok. Eventually, we'll remove this test that just prints as the evaluator matures. Then various ops will be unit tested via `Value` and integration tested with conformance/etc tests.